tfjs-backend-webgpu/src/kernels/FromPixels.ts

```diff
@@ -64,8 +64,10 @@ export function fromPixels(args: {
       [pixels.width, pixels.height];
   const outputShape = [height, width, numChannels];
 
+  // Disable importExternalTexture temporarily as it has problem in spec and
+  // browser impl
   const importVideo =
-      env().getBool('WEBGPU_IMPORT_EXTERNAL_TEXTURE') && isVideo;
+      false && env().getBool('WEBGPU_IMPORT_EXTERNAL_TEXTURE') && isVideo;
```

Comment:
Why not directly assign `WEBGPU_IMPORT_EXTERNAL_TEXTURE` to false in `flags_webgpu.ts`?

Reply:
I understand this disables import path completely.